src/terra-dev-site/guide/Internationalization/AddingTranslationsForALocaleNotSupportedByTerraUI.d.guide.md

```diff
@@ -0,0 +1,56 @@
+# Adding Translations For A Locale Not Supported By Terra UI
+
+Terra UI provides translations for [the following locales](https://github.com/cerner/terra-toolkit/blob/master/scripts/aggregate-translations/i18nSupportedLocales.js) for any strings we use in our components. In some cases, you may need to support locales outside of the ones we provide by default.
```

Comment:
for any strings we use in our -> for any string used in our

Reply:
Updated: https://github.com/cerner/terra-ui/pull/132/commits/515976f0d8690f2dfca2d7c5164ee8e9c46e468f